app/App.js

```diff
@@ -122,12 +122,13 @@ class App extends React.Component<Props, State> {
     });
   };
 
-  healthCheckFlow = () => {
-    const healthCheckIntervalTime = 60000;
-    store.dispatch(checkNodeConnection());
-    this.healthCheckInterval = setInterval(() => {
-      store.dispatch(checkNodeConnection());
-    }, healthCheckIntervalTime);
+  connectionAndMiningFlow = async () => {
```

Comment:
entering this function means that isConnected === true? If it is then is it necessary to check connectivity once again?